src/main/java/robot/subsystems/hatchIntake/commands/InsertHatch.java

```diff
@@ -0,0 +1,13 @@
+package robot.subsystems.hatchIntake.commands;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class InsertHatch extends CommandGroup {
```

Comment:
`PlaceHatch` would be a better name, because the Hatch Panels are placed on a wall and not inserted into it, like the Cargo.